app/services/inherited_proofing/va/service_payload_validator.rb

```diff
@@ -0,0 +1,74 @@
+require_relative 'field_validatable'
+
+module InheritedProofing
+  module Va
```

Comment:
Would it be possible to move the tooling for validation into its own PR? I think that may help keep the diffs small and easier to review and get merged up.

Reply:
I'll see what I can do.

Reply:
Mentioned [here](https://github.com/18F/identity-idp/pull/6608#discussion_r930170473), but I think it's preferable to use existing validation tools for consistency's sake 🙂

Reply:
> Mentioned [here](https://github.com/18F/identity-idp/pull/6608#discussion_r930170473), but I think it's preferable to use existing validation tools for consistency's sake 🙂

Of course. I understand :). We went down a path and I need to discuss it with my team, one of whom is not in ATM. We're at the end of the sprint today.

Reply:
@jmhooper, all that is being moved to this sprint, and will be using the standard AM paradigm.